qpc/report/tests_deprecated_report_detail.py

```diff
@@ -0,0 +1,219 @@
+#
+# Copyright (c) 2018 Red Hat, Inc.
+#
+# This software is licensed to you under the GNU General Public License,
+# version 3 (GPLv3). There is NO WARRANTY for this software, express or
+# implied, including the implied warranties of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE. You should have received a copy of GPLv3
+# along with this software; if not, see
+# https://www.gnu.org/licenses/gpl-3.0.txt.
+#
+"""Test the CLI module."""
+
+import json
+import os
+import sys
+import time
+import unittest
+from argparse import ArgumentParser, Namespace
+from io import StringIO
+
+from qpc import messages
+from qpc.cli import CLI
+from qpc.report import REPORT_URI
+from qpc.report.deprecated_detail import DeprecatedReportDetailCommand
+from qpc.scan import SCAN_JOB_URI
+from qpc.tests_utilities import DEFAULT_CONFIG, HushUpStderr, redirect_stdout
+from qpc.utils import get_server_location, write_server_config
+
+import requests_mock
+
+PARSER = ArgumentParser()
+SUBPARSER = PARSER.add_subparsers(dest='subcommand')
+
+
+class ReportDetailTests(unittest.TestCase):
+    """Class for testing the scan show commands for qpc."""
```

Comment:
I know that this was not introduced by you but change this docstring to reflect report commands

Reply:
😂